docs/content/docs/1.getting-started/5.contributing.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Contribution'
+description: '@nuxt/scripts is a community project - and so we love contributions of all kinds! ❤️'
+navigation.icon: i-ph-git-pull-request-duotone
+---
+
+::Callout
+ℹ Read more about contributions in the Nuxt eco-system [here](https://nuxt.com/docs/community/contribution)
+::
+
+
+## Preparing your environment
+
+`@nuxt/scripts` is a monorepo using pnpm workspaces. The package manager used to install and link dependencies must be pnpm.
+
+After cloning the repo, you can prepare your environments with
+
+```sh
+pnpm i
+pnpm dev:prepare
+```
```

Comment:
Hi @huang-julien , thanks for making the guide.

I have one suggestion. I'd like to hear your opinion.

#### Q. What about having `postinstall` scripts and not making contributors aware of `pnpm dev:prepare`?

```json
"scripts": {
"dev:prepare: "....",
"postinstall": "pnpm dev:prepare"
}
```

Reply:
make sense :+1: